docs/excel/custom-functions-json.md

```diff
@@ -122,7 +122,7 @@ The `options` object enables you to customize some aspects of how and when Excel
 |:-----|:-----|:-----|:-----|
 |  `cancelable`  |  boolean  |  No<br/><br/>Default value is `false`.  |  If `true`, Excel calls the `onCanceled` handler whenever the user takes an action that has the effect of canceling the function; for example, manually triggering recalculation or editing a cell that is referenced by the function. If you use this option, Excel will call the JavaScript function with an additional `caller` parameter. (Do ***not*** register this parameter in the `parameters` property). In the body of the function, a handler must be assigned to the `caller.onCanceled` member. For more information, see [Canceling a function](custom-functions-overview.md#canceling-a-function). |
 |  `stream`  |  boolean  |  No<br/><br/>Default value is `false`.  |  If `true`, the function can output repeatedly to the cell even when invoked only once. This option is useful for rapidly-changing data sources, such as a stock price. If you use this option, Excel will call the JavaScript function with an additional `caller` parameter. (Do ***not*** register this parameter in the `parameters` property). The function should have no `return` statement. Instead, the result value is passed as the argument of the `caller.setResult` callback method. For more information, see [Streaming functions](custom-functions-overview.md#streaming-functions). |
-|  `volatile`  | boolean | No <br/><br/>Default value is `false`. | If `true`, the function will recalculate each time Excel recalculates, instead of only when the formula's dependent values have changed. A function cannot be both streaming and volatile. If the `stream` and `volatile` properties are both set to `true`, the volatile option will be ignored. |
+|  `volatile`  | boolean | No <br/><br/>Default value is `false`. | **Coming to January Office Insiders Build** <br /><br /> If `true`, the function will recalculate each time Excel recalculates, instead of only when the formula's dependent values have changed. A function cannot be both streaming and volatile. If the `stream` and `volatile` properties are both set to `true`, the volatile option will be ignored. |
```

Comment:
If this is now in Jan - does this mean that the docs should take it out until Jan or do you want people to know in advance for some reason? Does including it currently break things? My understanding the last time I tried it was that it is ok except that the intelliSense doesn't work for it yet.

Reply:
Checked with Reeza, I will update to take out and then merge this PR